docs/management/hosts/hosts-overview.asciidoc

```diff
@@ -1,47 +1,33 @@
 [[hosts-overview]]
-= Hosts page overview
-The Hosts page provides a comprehensive overview of all hosts and host-related security events. Key Performance Indicator (KPI) charts, data tables, and interactive widgets let you view specific data, drill down for deeper insights, and interact with the Timeline Event Viewer for further investigation.
+= Hosts page
+
+The Hosts page provides a comprehensive overview of all hosts and host-related security events. Key performance indicator (KPI) charts, data tables, and interactive widgets let you view specific data, drill down for deeper insights, and interact with Timeline for further investigation.
 
 [role="screenshot"]
 image::images/hosts-ov-pg.png[Hosts page]
 
-[float]
-*Search and filter hosts*
-
-The *{kibana-ref}/kuery-query.html[{kib} Query Language (KQL)]* bar, which appears at the top of each page in the Elastic Security app, is useful to search and filter hosts. Use the timepicker to the right of the KQL bar to select a specific date and time range.
-NOTE: For more information about searching using KQL, see {kibana-ref}/search.html[Search data].
+*Host KPI charts*
 
-*Host Key Performance Indicators (KPIs) charts*
+KPI charts show metrics for hosts and unique IPs within the time range specified in the date picker. Data in the KPI charts is depicted via linear or bar graphs.
 
-KPI charts show data metrics for hosts, user authentications, and unique IPs based on the time range specified in the date picker. Data in the KPI charts is depicted via linear or bar graphs.
 NOTE: The default time range is within the last 15 minutes.
 
 *Data tables*
 
 Beneath the KPI charts are data tables, categorized by individual tabs, which are useful for viewing and investigating specific types of data. Select the relevant tab to view the following data:
 
-* *All hosts*: High-level host details
-* *Authentications*: Authentication events
-* *Uncommon processes*: Uncommon processes running on hosts
-* *Anomalies*: Anomalies discovered by machine learning jobs
-* *Events*: All host events
-* *External alerts*: Alerts received from external monitoring tools
-
-Numerous values in the data tables (e.g., host name, process name, etc.) contain inline options when you hover over them: *Filter for value*, *Filter out value*, *Add to timeline investigation*, *Show top <field name>*, and *Copy to Clipboard*.
-
-[role="screenshot"]
-image::images/inline-actions.png[Inline actions in the data table]
-
-*Add data values to the Timeline*
+* *All hosts*: High-level host details.
+* *Uncommon processes*: Uncommon processes running on hosts.
+* *Anomalies*: Anomalies discovered by machine learning jobs.
+* *Events*: All host events.
+* *External alerts*: Alerts received from external monitoring tools.
+* *Sessions*: Linux process events that you can open in Session View, an investigation tool that allows you to examine Linux process data at a hierarchal level.
```

Comment:
Did you want to link to the Session View docs here?

Reply:
Session View topic is now merged, so should be good to link.